lib/components/src/app-shell/app-shell.stories.js

```diff
@@ -138,12 +138,11 @@ const Template = ({ navClick }) => {
     </td-list>
 
     <div slot="mini-list">
-            <div class="td-toolbar">
-                <span class="mdc-typography--headline6">[Page name]</span>
-                <div class="spacer"></div>
-                <td-icon-button slot="actionItems" icon="filter_list"></td-icon-button>
-                <td-icon-button slot="actionItems" icon="add"></td-icon-button>
-            </div>
+            <td-toolbar>
+                <span class="mdc-typography--headline6" slot="title">[Page name]</span>
+                <td-icon-button slot="iconActions" icon="filter_list"></td-icon-button>
+                <td-icon-button slot="iconActions" icon="add"></td-icon-button>
+            </td-toolbar>
             <div divider></div>
```

Comment:
This should be done with CSS on the bottom of the toolbar itself, not in the markup

Reply:
For the `<div divider></div>`?

Reply:
Yup. It's purely stylistic not functional so should go into the styles